packages/app-project/src/screens/ClassifyPage/ClassifyPageContainer.spec.js

```diff
@@ -9,6 +9,7 @@ import sinon from 'sinon'
 import { expect } from 'chai'
 
 describe('Component > ClassifyPageContainer', function () {
+	this.timeout(10000)
```

Comment:
This seems out of place for the WidgetHeading RTL conversion, how is it related? Otherwise everything else LGTM.

Reply:
The classifier tests were timing out with the changes. This simply enabled all the tests to run - it doesn't affect the actual component itself, just how long the test can run before timing out.

Reply:
Sorry, I didn't realize this is holding on me!

I remember CI tests were failing in the spring/summer because the tests were taking a long time to run. I think there were some refactors subsequent to this PR that improved the CI tests timing. Is this still necessary?

When I run `yarn test:ci` locally after removing this line the tests are "Done in 42.26s" which seems reasonable to me.